api/capi/src/nnstreamer-capi-single.c

```diff
@@ -116,6 +117,12 @@ ml_single_open (ml_single_h * single, const char *model,
     }
   }
 
+  if (!model || !g_file_test (model, G_FILE_TEST_IS_REGULAR)) {
+    ml_loge ("The given param, model path [%s] is invalid.",
+        GST_STR_NULL (model));
+    return ML_ERROR_INVALID_PARAMETER;
+  }
```

Comment:
What if ```enable_symlink=TRUE```?

Reply:
It is ok to run symlink model file. 
'enable_symlink' option is no effect to model file, do we need to handle symlink option for the NN model?

Reply:
I thought G_FILE_TEST_IS_REGULAR will fail if model is symlink... But according to https://developer.gnome.org/glib/stable/glib-File-Utilities.html, it is ok. So please never mind.